plugins/wpe-headless/includes/replacement/functions.php

```diff
@@ -20,11 +20,11 @@ function wpe_headless_domain_replacement_enabled() {
 	$enabled = false;
 
 	if ( wpe_headless_is_rewrites_enabled() ) {
-		if ( isset( $_GET['replace-domain'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
The removed `phpcs:ignore` comment seems to be the cause of the CI failure here.